js/libs/ui-shared/src/context/environment.ts

```diff
@@ -19,6 +19,7 @@ export type BaseEnvironment = {
   logo: string;
   /** The URL to be followed when the logo is clicked. */
   logoUrl: string;
+  scope?: string;
```

Comment:
If this is only used in the [account environment](https://github.com/keycloak/keycloak/blob/ced8e643c9c80ccd549439d0db9d03088748b464/js/apps/account-ui/src/environment.ts#L6). Ideally this field is also commented so folks know what it does.